### PR TITLE
Create 2019-12-28-countdown-to-the-new-year-ember-changeset.md

### LINKED ISSUE #465: Create blog post for December 28, 2019 about the `ember-changeset` addon
## Goal
Write a short blog post for the December blog series that is similar to the [example post]( https://github.com/ember-learn/ember-blog/blob/be8f29ba0ae8b5367d1758b1a150c1b503812a77/source/2019-12-01-countdown-to-the-new-year-ember-sortable.md). You can find more information on how to write your post in our [Contribution guidelines for DecEmber](https://github.com/ember-learn/ember-blog/blob/master/source/december-2019-blog-series-template.md).

## Criteria

- [ ] file name should 2019-12-DD-countdown-to-the-new-year-ADDON-NAME.md where DD is replaced with the day and ADDON-NAME is replaced with the hyphenated name of the addon
- [ ] The opening sentence should be updated where COUNT is the sequential ordinal word. 

```
**This is the COUNT in our DecEmber series- "Countdown to The New Year: 31 Days of 
Ember Addons".  We plan to highlight a new addon each day until the new year, and 
we hope you'll join us for the fun!**
```

- [ ] The first h2 should be "Day NUM" where NUM is the same as the DD in the date.
- [ ] the link to the addon should point to the page on Ember Observer
- [ ] There should be two sub sections: (both h3)- "What it does" and "Why I like it" 
- [ ] Include whether or not the addon has been updated yet for Ember Octane- if not, invite contributors to help get the addon Octane ready.
- [ ] The closing sentence can be slightly altered but should feel similar: 

```
"Do you use this addon? Or one like it? We'd love to hear about Ember addons that bring you joy!"
```

### PR DESCRIPTION
**Pull request by snewcomer**
close #465

### CODE REVIEW

**Review by MelSumner — APPROVED** [review]
This looks great, thank you for writing it. We'll merge it for publishing on the 28th. :+1:

File: source/2019-12-28-countdown-to-the-new-year-ember-changeset.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+**This is the 28th in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 28
+
+For the 28th edition of **Countdown to The New Year** we're going to take a
+look at [`ember-changeset`](https://emberobserver.com/addons/ember-changeset).
+
+### Ember Changeset - What It Does
```
Comment (jayjayjpg) [inline source/2019-12-28-countdown-to-the-new-year-ember-changeset.md]: I think for the December series title formatting, we currently use the following - what do you think?

```suggestion
### What It Does

```

File: source/2019-12-28-countdown-to-the-new-year-ember-changeset.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Managing data associated with forms can sometimmes be unwieldy. As a result, we have borrowed concepts from the Elixir [community](https://hexdocs.pm/ecto/Ecto.Changeset.html#content) to give you the ability to prevent mutating the underlying object until the last possible moment when you have decided to apply the changes and the data is valid. This functional approach to validations has proved to be quite useful.
+
+This addon works in conjunction with [`ember-changeset-validations`](https://github.com/poteto/ember-changeset-validations/) to provide you with out of the box validations to validate the format of a certain input, ensure passwords pass a minimum security threshold or even let you build your own custom validator!
+
+### Ember Changeset - Why I Like It
```
Comment (jayjayjpg) [inline source/2019-12-28-countdown-to-the-new-year-ember-changeset.md]: What do you think about the following? This should resolve the markdown lint error on CI:

```suggestion
### Why I Like It

```

File: source/2019-12-28-countdown-to-the-new-year-ember-changeset.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+Recently I [rewrote](https://github.com/poteto/ember-changeset/pull/379) `ember-changeset` to support Octane patterns and utilized `@tracked`.  As a result, not only can you validate and apply changes to top level keys (`person`), nested keys such as `person.firstName` are now easily possible.  In addition, at the same time `@pzuraq`'s blog post [Do You Need EmberObject?](https://www.pzuraq.com/do-you-need-ember-object/) came out, we removed `EmberObject`, giving you a simpler base class to work with.  Currently this functionality is still in `v3` beta.  We plan to exit the beta period and do a proper `3.0.0` release in the near future for apps supporting `ember-source` >= 3.13.
```
Comment (jayjayjpg) [inline source/2019-12-28-countdown-to-the-new-year-ember-changeset.md]: What do you think about the following wording instead to resolve the language check error?

```suggestion
Recently I [rewrote](https://github.com/poteto/ember-changeset/pull/379) `ember-changeset` to support Octane patterns and utilized `@tracked`.  As a result, not only can you validate and apply changes to top level keys (`person`), nested keys such as `person.firstName` may also be used.  In addition, at the same time `@pzuraq`'s blog post [Do You Need EmberObject?](https://www.pzuraq.com/do-you-need-ember-object/) came out, we removed `EmberObject`, giving you a simpler base class to work with.  Currently this functionality is still in `v3` beta.  We plan to exit the beta period and do a proper `3.0.0` release in the near future for apps supporting `ember-source` >= 3.13.
```

**Review by jayjayjpg — APPROVED** [review]
This reads great! Currently the linter checks on CI error with the following:

```
source/2019-12-28-countdown-to-the-new-year-ember-changeset.md
  22:279-22:285  warning  `easily` may be insensitive, try not to use it  easy  retext-equality
⚠ 1 warning
The command "bash scripts/language-check.sh" exited with 1.
0.60s$ bash scripts/markdown-lint.sh
fatal: remote base already exists.
source/2019-12-28-countdown-to-the-new-year-ember-changeset.md:16 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "### Ember Changeset - What It Does"]
source/2019-12-28-countdown-to-the-new-year-ember-changeset.md:21 MD022/blanks-around-headings/blanks-around-headers Headings should be surrounded by blank lines [Expected: 1; Actual: 0; Below] [Context: "### Ember Changeset - Why I Like It"]
```

I applied a few suggestions to resolve the CI errors, feel free to apply any you find useful or rephrase on your own. Last but not least, thank you for this great writeup for December :sparkles:

**Review by MelSumner — APPROVED** [review]

**Pull request merged**